src/main/java/uk/gov/hmcts/ccd/security/filters/PathParamSecurityFilter.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.ccd.security.filters;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.ccd.security.exception.UnauthorizedException;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.function.Function;
+
+@Component
```

Comment:
Where is this bean instantiated?
I cant seem to find that anywhere in this code

Reply:
https://github.com/hmcts/ccd-data-store-api/blob/9fdc0723f8c35c21b6d73e9da9d52fc5c28cdd7b/src/main/java/uk/gov/hmcts/ccd/AuthCheckerConfiguration.java#L51-L52